### PR TITLE
DOCS-#3734: remove obvious sentences from dispatcher docstrings

### LINKED ISSUE #3734: remove obvious sentences from dispatchers docstrings
For example: `Inherits some common...`.

Initial found in https://github.com/modin-project/modin/pull/3537#discussion_r737664091.

### PR DESCRIPTION
**Pull request by anmyachev**
Signed-off-by: Anatoly Myachev <anatoly.myachev@intel.com>

## What do these changes do?

- [ ] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [ ] passes `flake8 modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [ ] passes `black --check modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [ ] signed commit with `git commit -s` 
- [x] Resolves #3734 
- [ ] tests added and passing
- [ ] module layout described at `docs/developer/architecture.rst` is up-to-date

### CODE REVIEW

**Review by YarShev — COMMENTED** [review]
I would prefer to extend docstrings for classes, instead of reducing them.

File: modin/core/io/pickle/pickle_dispatcher.py [hunk 1]
```diff
@@ -21,11 +21,7 @@
 
 
 class PickleExperimentalDispatcher(FileDispatcher):
-    """
-    Class handles utils for reading `.pkl` files.
-
-    Inherits some common from `FileDispatcher` class.
-    """
+    """Class handles utils for reading `.pkl` files."""
```
Comment (vnlitvinov) [inline modin/core/io/pickle/pickle_dispatcher.py:24]:
```suggestion
    """Class handles utils for reading pickle files."""
```
I don't think `.pkl` is the standard extension, unlinke `.csv` or `.parquet`.

**Review by vnlitvinov — APPROVED** [review]
LGTM with minor nit pick.

**Comment by YarShev** [comment]
> I would prefer to extend docstrings for classes, instead of reducing them.

@vnlitvinov , thoughts?

**Comment by vnlitvinov** [comment]
> > I would prefer to extend docstrings for classes, instead of reducing them.
> 
> @vnlitvinov , thoughts?

In general I would agree, but stating "this inherits common features from Foo" where `Foo` is plainly stated as a parent class is "thank you, captain Obvious" kind of useless thing.

I mean, I'm all in for bigger docstrings, but they should be bigger by bringing value, not by re-phrasing `class Bar(Foo)` statement.

**Review by vnlitvinov — APPROVED** [review]
LGTM

**Comment by YarShev** [comment]
At some point we may want to generate docs for those objects. By default sphinx doesn't expose base classes and those lines of docstrings might be helpful. An alternative option might be `show-inheritance` to use in such a case. So we'll probably be relying on the latter.

**Review by YarShev — APPROVED** [review]

**Pull request merged**